cordformation/src/main/kotlin/net/corda/plugins/ConfigurationUtils.kt

```diff
@@ -0,0 +1,19 @@
+package net.corda.plugins
+
+import org.gradle.api.InvalidUserDataException
+import java.net.URI
+import java.net.URISyntaxException
+
+class ConfigurationUtils {
+
+    companion object {
+
+        fun parsePort(address: String): Int {
+            return try {
+                URI(null, address, null, null, null).port
+            } catch (ex: URISyntaxException) {
+                throw InvalidUserDataException("Invalid host and port syntax for RPC address, expected host:port. Using default value")
```

Comment:
This exception will kill Gradle, which is fine. It's just that the message shouldn't therefore say that the default value will be used.